### ISSUE TITLE
BioDT UiO team meeting schedule

### ISSUE THREAD

**Issue by dagendresen** [uio-mana/BioDT#16]
[Since February 2023 we have been trying to meet bi-weekly on Wednesday morning at 10:00](https://docs.google.com/document/d/1C4fHZjyCLOUQY4BX2-sU7UKT1xbYzG5IOpghK8hUtto/edit).

Erik is working 50% at UiO and the BioDT project from September 2024 and will not be available on Wednesday mornings. @ErikKusch the Outlook calendar displays the days you plan to be at Tøyen, but the days you have marked seems to vary by different weeks? Fridays seems to be the most stable day?

Do we want to continue to meet bi-weekly? Or maybe change to monthly?
If all of us have updated calendars I could schedule bi-weekly (monthly?) meetings when we are all free.

**Comment by cjandrew** [comment]
I would vote for a monthly meeting over bi-weekly

**Comment by ErikKusch** [comment]
I agree with @cjandrew that monthly meetings of 1-hour may be more useful than bi-weekly meetings of 0.5h duration. At least for me, that would make scheduling a lot easier. Here is what I suggest:

BioDT Meetings:
2024-09-27
2024-10-25
2024-11-22

MANA Meetings:
2024-10-11 (I already have holidays booked for this day, I am afraid)
2024-11-08
2024-12-06

With this schedule, we still see each other bi-weekly, but have BioDT and MANA meetings rotating on a 4-week basis. Thoughts?

**Comment by cjandrew** [comment]
Good suggestion, @ErikKusch, but what do you think about condensing the meetings even further? 

I would prefer 45-minute monthly meetings of both MaNa and BioDT, on the same day, with 15-30 minutes break in-between. It would be a total of 1.75-2 hours, but then the rest of the month is freed up, and it would provide better transitional space for overlapping topics.

So, I will suggest another option: 
First meet BioDT (e.g., 10:15-11:00), then meet MaNa (e.g., 11:15-12:00). Just once a month.
2024-09-27
2024-10-25
2024-11-22
2024-12-20 (jul middag?)

Thoughts, all?

**Comment by dagendresen** [comment]
Meeting less frequently is good for freeing up time. But I also need an update from each of you on progress. I suggest that all of you send a weekly brief report with an update on what you are up to and working on. Maybe make this a rolling report in a Google document?

**Comment by cjandrew** [comment]
I think we should be honest: Next to never in these meetings (MaNa or BioDT) have we consistently gone around the table and updated each other on our work, so I do not see the correlation between meeting frequency and us reporting on work. Normally one person will say something, and topics will diverge, or the absence of an individual will be reason to not update those actually present. People are invited to speak up (few ever do), but each are not individually requested to report on their work.

Weekly updates is a totally different subject than the current one, on whether two meetings are combined to monthly versus interspersed bi-weekly. The way in which you bring up the weekly reporting, to me, signifies a lack of trust on your part in what we are doing. I find this disappointing, but I would follow along with your new initiative, however you arrange it. For me, discussing when meetings does not impact my weekly work. And I don't appreciate the insinuation that the two are correlated. 👎

**Comment by ErikKusch** [comment]
I do agree with @cjandrew that neither MANA meetings nor BioDT meetings have served as a platform where work reports are done from everyone. I remember that, when I pushed us into having 0.5h meeting slots, I did suggest that we spend 5 minutes at the beginning of each meeting to update each other on what has been happening on our ends and what we would focus on next. 

I also agree with @cjandrew that we now have two separate topics here: (1) new meeting schedule and (2) work reporting. Regarding the former, I agree with @cjandrew's [proposed schedule](https://github.com/uio-mana/PROJECT-BioDT/issues/16#issuecomment-2363004175) above. With regard to the latter, I fully understand that @dagendresen needs to have an overview of our work. I personally do not see this as a lack of trust, but rather assume this comes down to reporting that Dag has to do further up the chain. As a result, I am happy to put weekly reports in via a google doc. Maybe this would best be discussed in-person at the proposed meeting next week?

Lastly, @cjandrew, @dagendresen, @DesalegnChala, @MichalTorma, @rukayaj  - please leave a reaction on this comment here as either a thumbs-up or thumbs-down whether the schedule suggested by Carrie works for you. I will then do the room booking and send RSVPs.

**Comment by cjandrew** [comment]
I like your interpretation, @ErikKusch, that weekly updates are more about reporting upwards than about a sudden lack of trust. However, why, then, have we not been doing that already, and why did the topic come up with reference to meeting frequency? These are rhetorical questions I write (not intending a response), but presented to explain why I have questioned the motives. 

Moving on to not repeating such conflict in the future: Possibly adding more context to reasons for such suggestions, @dagendresen (but also anyone else, including me), in the future could help clarify and reduce confusion?

**Comment by dagendresen** [comment]
I want all of BioDT project staff to collaborate more and believe there is a need for more communication across the team. I also need to report achievements and progress on each BioDT task including at the EC review meeting on Wednesday next week. It is unclear to me why some might seem to be reluctant to collaborate together to deliver our responsibilities in the BioDT project. We can aim at solving cross-team communication in regular in-person meetings, or we can try to make a system for informing each other in other ways - such as a brief weekly report.

**Comment by cjandrew** [comment]
We should have two threads, like @ErikKusch wrote. Anyways, then, @dagendresen  you should know that I've already sent an updated Task 5.3 slide to Sharif.

@dagendresen, this makes no sense to me, so please clarify if it is relevant (otherwise just leave it): "It is unclear to me why some might seem to be reluctant to collaborate together to deliver our responsibilities in the BioDT project." In my impression, people are working on the responsibilities and collaborating when needed.

[
[BIODT-EC-review2024-WP5_Task5-3_CA-20sept2024.pptx](https://github.com/user-attachments/files/17073271/BIODT-EC-review2024-WP5_Task5-3_CA-20sept2024.pptx)
](url)

**Comment by ErikKusch** [comment]
Calendar invitations according to [@cjandrew's proposal](https://github.com/uio-mana/PROJECT-BioDT/issues/16#issuecomment-2363004175) are sent. I would suggest we talk about progress reporting and cross-team collaboration next week at our meeting?

**Comment by DesalegnChala** [comment]
A compromise could be providing updates when significant developments occur between meetings. This ensures important information isn't delayed while avoiding unnecessary reports. Dag can also request updates when needed, such as when preparing his reports. Just my two cents but I agree better to discuss at the next meeting.
